src/test/consensus/LedgerTrie_test.cpp

```diff
@@ -348,36 +362,43 @@ class LedgerTrie_test : public beast::unit_test::suite
         using namespace csf;
         using Seq = Ledger::Seq;
         // Empty
+        {
+            LedgerTrie<Ledger> t;
+            BEAST_EXPECT(t.getPreferred(Seq{0}) == boost::none);
+            BEAST_EXPECT(t.getPreferred(Seq{2}) == boost::none);
+        }
+        // Genesis support is NOT empty
         {
             LedgerTrie<Ledger> t;
             LedgerHistoryHelper h;
-            BEAST_EXPECT(t.getPreferred(Seq{0}).id == h[""].id());
-            BEAST_EXPECT(t.getPreferred(Seq{2}).id == h[""].id());
+            Ledger genesis = h[""];
+            t.insert(genesis);
+            BEAST_EXPECT(t.getPreferred(Seq{0})->id == genesis.id());
```

Comment:
```suggestion
            BEAST_EXPECT(t.getPreferred(Seq{0})->id == genesis.id());
            BEAST_EXPECT(t.remove(genesis));
            BEAST_EXPECT(t.getPreferred(Seq{0}) == boost::none);
            BEAST_EXPECT(!t.remove(genesis));
```
...This may be redundant with `testEmpty`, but it really helps clarify for me what's going on.